modules/ROOT/pages/bare-metal.adoc

```diff
@@ -19,14 +19,14 @@ To install FCOS onto bare metal with an ISO, follow these steps:
 NOTE: You can install in either legacy boot (BIOS) mode or in UEFI mode, regardless of what mode the OS will use once installed.
 +
 . Burn the ISO to disk and boot it.
-. From the boot menu, press `<TAB>` (isolinux) or `e` (grub) to edit the kernel command line.
-. Add the following parameters to the kernel command line:
-* `coreos.inst.install_dev=/dev/sda`
-* `coreos.inst.stream=stable`
-* `coreos.inst.ignition_url=http://example.com/config.ign`
-. Press `<ENTER>` (isolinux) or `<CTRL-x>` (grub) to start the installation.
-
-Once the installation is complete, the machine reboots. After rebooting, the first boot process begins. It is at this time that Ignition ingests the configuration file and provisions the system as specified.
+. Once booted, you can fetch your Ignition config and run `coreos-installer`:
+[source, bash]
+----
+curl -LO http://example.com/example.ign
+sudo coreos-installer install /dev/sda --ignition example.ign
+----
+
```

Comment:
It might be worth adding a **Tip** for running `coreos-install install --help` to explore more options that can be passed to the installer.